catalog-info.yaml

```diff
@@ -2,6 +2,7 @@ apiVersion: backstage.io/v1alpha1
 kind: Component
 metadata:
   name: twiglet
+  title: Twiglet
```

Comment:
I think we should make the name match the repo, what do you think?

Reply:
Yeah, good idea